test/parallel/test-fs-utimes.js

```diff
@@ -122,16 +122,30 @@ function runTest(atime, mtime, callback) {
 
 var stats = fs.statSync(__filename);
 
+
+// toUnixTimestamp
+assert.equal(fs._toUnixTimestamp('12345'), 12345);
+assert.equal(fs._toUnixTimestamp(NaN), Date.now() / 1000);
+assert.equal(fs._toUnixTimestamp(Infinity), Date.now() / 1000);
+assert.equal(fs._toUnixTimestamp(-1), Date.now() / 1000);
```

Comment:
I don't cache the `Date.now()` just because they should be in the same one loop. So the value should be equal.